Content.Server/Glue/GlueSystem.cs

```diff
@@ -77,13 +77,16 @@ public override void Update(float frameTime)
     {
         base.Update(frameTime);
 
-        var query = EntityQueryEnumerator<GluedComponent, UnremoveableComponent>();
-        while (query.MoveNext(out var uid, out var glue, out _))
+        var query = EntityQueryEnumerator<GluedComponent, MetaDataComponent, UnremoveableComponent>();
+        while (query.MoveNext(out var uid, out var glue, out var meta, out _))
         {
```

Comment:
```suggestion
        var query = EntityQueryEnumerator<GluedComponent, UnremoveableComponent, MetaDataComponent>();
        while (query.MoveNext(out var uid, out var glue, out var meta, out _))
        {
```

In-engine atm you want least frequent to most frequent.

Reply:
Ahhh yes I see, it narrows from left to right.